cli/include/hse/cli/rest/client.h

```diff
@@ -23,8 +26,7 @@ rest_client_init(const char *socket_path);
  * hsettp tool.
  */
 merr_t
-rest_client_fetch(
-    const char *method,
+rest_client_fetch(const char *method,
```

Comment:
I don't understand what is going on here?

Reply:
I don't know exactly what combination of settings caused this, but it's the lesser of 25 evils IMO.

Reply:
It was way better before. This is like the most evil of all 25 evils :).

Reply:
soon there will be nothing left of my tongue..

Reply:
I will investigate in a bit.